src/libDirectoryService/FinalBlockPostProcessing.cpp

```diff
@@ -170,6 +170,10 @@ void DirectoryService::ProcessFinalBlockConsensusWhenDone() {
         << m_mediator.m_txBlockChain.GetLastBlock().GetHeader().GetBlockNum() +
                1
         << "] FINISH WRITE STATE TO DISK");
+    if (m_mediator.m_dsBlockChain.GetLastBlock().GetHeader().GetBlockNum() <
+        PREGEN_ACCOUNT_TIMES) {
```

Comment:
This variable means PREGEN accounts before this block number? Maybe should change to PREGEN_ACCOUNT_BLOCK_NUM?

Reply:
It means the time pregen has been called. Block num is just used as a reference here